src/qml/filters/audio_normalize_1p/ui.qml

```diff
@@ -27,6 +27,7 @@ Item {
         maxgainSlider.value = filter.getDouble('max_gain');
         mingainSlider.value = filter.getDouble('min_gain');
         maxrateSlider.value = filter.getDouble('max_rate');
+        discResetCheckbox.checked = filter.get('discontinuity_reset');
```

Comment:
You cannot set a boolean from this property value directly. In JavaScript a "0" is true:
```
> !!"0"
true 
```
You can use `parseInt(filter.get())`.

Reply:
Done